### PR TITLE
Millstone: New York Times clone

### PR DESCRIPTION
**Pull request by lucianosarno**
In this project we were required to create a clone of NYT website (https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0).

### CODE REVIEW

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @lucianosarno  and @MohameDawood ! 👋 
Great job with this NYT clone! It looks really good so far👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****
- [x] Your pull request has a great title, however, there is no description. **It is extremely important that you add a short description of what you have done in your pull request.** Please add this before requesting for your next review and for **all future reviews.** 

- [x] There is no live demo link in your README file. Please add one. You can generate this using [this link](https://raw.githack.com/).

- [x] The `More in space and astronomy` and `Editor's picks` articles should be wrapped in link tags. 
Screen Shot 2020-06-23 at 10 56 51 PM

- [x] The navigation bar should stick to the top of the screen, yours does not. Please fix this. 
Screen Shot 2020-06-23 at 11 02 46 PM

- [x] Please remove all commented code. It makes your project look untidy. Please see my comments below. 
Screen Shot 2020-06-23 at 11 04 56 PM




****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,733 @@
...
+  font-size: 40px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 1rem;
+  font-stretch: 100%;
+  line-height: 48px;
+}
+
+svg {
+  width: min-content; /* height: min-content; */
```
Comment (BrittanyBlake) [inline css/styles.css]: please remove commented code

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,733 @@
...
+  line-height: 22px;
+}
+
+.more h2 {
+  border-top: 1px solid black;
+  padding: 17px 0 0 0;
+}
+
+.more-2 {
+  width: 314px; /* height: 100px; */
```
Comment (BrittanyBlake) [inline css/styles.css]: please remove commented code

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi again @lucianosarno  and @MohameDawood! 👋 
Great job adding a description to your pull request! 👏 👏  This however appears to be the only change that has been made. 
****
Please ensure to read my previous review and make the changes. Here is a shortlist of what was requested:

- [x] Add a live demo link to your Readme file which can be generated using githack.

- [x] Wrap the articles in `More in space and astronomy and editors picks` in `<a>` tags as they are links to other articles.

- [x] Make the navigation bar fixed. 
Your clone 👇 
Screen Shot 2020-06-23 at 11 50 52 PM
The original page 👇 
Screen Shot 2020-06-23 at 11 51 17 PM

- [x] There should also be a fixed footer on the page. I'm sorry that I did not point this out to you in the first review, I am not sure how I missed that 🤦🏾‍♀️
The original page 👇 
Screen Shot 2020-06-23 at 11 53 29 PM

Your clone 👇 
Screen Shot 2020-06-23 at 11 54 09 PM

- [x] Remove commented code



****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: css/styles.css (hunk 1 above)
Comment (BrittanyBlake) [reply css/styles.css]: Commented code has not yet been removed

File: css/styles.css (hunk 2 above)
Comment (BrittanyBlake) [reply css/styles.css]: Commented code has not yet been removed

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon:

Hi @lucianosarno and @MohameDawood, great job with the fixes :+1: Having fixed all the comments made by the previous TSE, I hereby approve your project. But I will leave an optional comment so you can optionally address. :smile:
[OPTIONAL]
- Kindly add a link to the NYT logo
- Kindly remove all angle tags ``` < ``` in the text, it makes the page unprofessional
- Fix the broken screenshot and address the description of the README. ```M=Dawood``` is not clear to me
- Add margin to the login button

Good luck with your next projects :+1:
_Happy coding_ :code:
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**